otel-agent/examples/otel-demo-docker-compose/docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
 
   # Collector
   otelcol:
-    image: otel/opentelemetry-collector-contrib:0.56.0
+    image: otel/opentelemetry-collector-contrib:0.72.0
```

Comment:
Why not latest release?

Reply:
I stand corrected! I was able to update to newer versions!

Reply:
0.75 is the latest image, even though newest source code is 0.76. Confirmed through testing

```                                                                                                                                              1.6s
Error response from daemon: manifest for otel/opentelemetry-collector-contrib:0.76.0 not found: manifest unknown: manifest unknown
```

Reply:
There were some issues with the latest release AFAIK, the correct image should be `0.76.1`.